core/pom.xml

```diff
@@ -42,6 +42,8 @@ THE SOFTWARE.
     <stapler.version>1.243</stapler.version>
     <spring.version>2.5.6.SEC03</spring.version>
     <groovy.version>2.4.7</groovy.version>
+    <!-- Currently there is a huge number of errors => we cannot fix all of them immediately -->
+    <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
Required to allow overriding in a profile (see [line 909](https://github.com/jenkinsci/jenkins/pull/2458/files/98b3620a0534beaa469120d5699209c52b5fa9f3#diff-357e4854869b2e21c38b1b437f11095aR909))